contacktbook/DataBase.cs

```diff
@@ -38,6 +43,8 @@ public void updateContact(Contact contact)
             using( var connection = new SQLiteConnection(connString))
             {
                 connection.Open();

```

Comment:
Sorry, I was wrong when told you that we shouldn't use connection.Open() in using statement. Using serves only for disposing object (connection in our case) as soon as it goes out of scope (close curly brace in our case)))